src/ui/layout/search/GrandSearch.vue

```diff
@@ -95,6 +95,11 @@ export default {
         },
         getPathsForObjects(objectsNeedingPaths) {
             return Promise.all(objectsNeedingPaths.map(async (domainObject) => {
+                if (!domainObject) {
+                    // user interrupted search, return back
+                    return null;
```

Comment:
can we add coverage for this negative test?

Reply:
I think this is being caught be our search/couch e2e test now, right @ozyx?

Reply:
That's a good catch. We will need to come up with a way to include that suite in our line coverage. I've put a note in our larger, follow-up couchdb testing issue here https://github.com/nasa/openmct/issues/5686

Reply:
> I think this is being caught be our search/couch e2e test now, right @ozyx?

Yes, it should be. Might not have been registered as being covered because we didn't trigger the couchdb e2e tests to run against this PR?

Reply:
Yeah, if it didn’t run, probably didn’t accumulate code coverage.